pkgs/development/python-modules/pdfrw/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, lib, buildPythonPackage, fetchFromGitHub, fetchPypi, isPyPy, python3, pytest, zlib }:
+
+with python3.pkgs;
+
+buildPythonPackage rec {
+  pname = "pdfrw";
+  version = "0.4";
+  name = "${pname}-${version}";
```

Comment:
Since a while back python packages default to `name = pname + "-" + version;` so this line is not necessary if `pname` and `version` are defined.